model/project_parser.go

```diff
@@ -1259,7 +1259,7 @@ func evaluateBVTasks(tse *taskSelectorEvaluator, tgse *tagSelectorEvaluator, vse
 				// it's already in the new list, so we check to make sure the status definitions match.
 				if !reflect.DeepEqual(t, old) {
 					evalErrs = append(evalErrs, errors.Errorf(
-						"conflicting definitions of build variant tasks '%s': %v != %v", name, t, old))
+						"conflicting definitions of task '%s' listed under build variant '%s': %#v != %#v", name, pbvt.Name, t, old))
```

Comment:
%#v will also print the package name, I don't know if that's helpful here.

Reply:
it would be `model.parserTask{ fields }` here

Reply:
The package/struct name isn't helpful, but the other formatters are less than ideal in their own ways, so I picked the one with the most field-level user-readable information. The only other alternative is `%+v` which is less human-readable.

Reply:
It seems like %+v would yield {x:1 y:2} and  %#v which would yield model.parserTask{x:1, y:2}.